lib/msal-common/test/client/OnBehalfOfClient.spec.ts

```diff
@@ -92,244 +123,48 @@ describe("OnBehalfOf unit tests", () => {
         });
     });
 
-    it("Does not add headers that do not qualify for a simple request", (done) => {
-        // For more information about this test see: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-        sinon.stub(OnBehalfOfClient.prototype, <any>"getCachedAuthenticationResult").resolves(null);
-        sinon.stub(OnBehalfOfClient.prototype, <any>"executePostToTokenEndpoint").callsFake((tokenEndpoint: string, queryString: string, headers: Record<string, string>) => {
-            const headerNames = Object.keys(headers);
-            headerNames.forEach((name) => {
-                expect(CORS_SIMPLE_REQUEST_HEADERS).toEqual(expect.arrayContaining([name.toLowerCase()]));
-            });
-
-            done();
-            return AUTHENTICATION_RESULT_DEFAULT_SCOPES;
-        });
-        
-        const client = new OnBehalfOfClient(config);
-        const onBehalfOfRequest: CommonOnBehalfOfRequest = {
-            authority: TEST_CONFIG.validAuthority,
-            correlationId: TEST_CONFIG.CORRELATION_ID,
-            scopes: TEST_CONFIG.DEFAULT_GRAPH_SCOPE,
-            oboAssertion: TEST_TOKENS.ACCESS_TOKEN,
-            skipCache: false
-        };
-
-        client.acquireToken(onBehalfOfRequest);
```

Comment:
Why were these tests deleted?

Reply:
@bmahall reorganized some tests. I added the tests for the `filtering` regarding `access_token`.